changelog.md

```diff
@@ -1,3 +1,23 @@
+3.1.0
+=========
+- Updated to Microsoft.IdentityModel.* 8.0.2
+
+### Security improvement:
+- Id Web now uses `CaseSensitiveClaimsIdentity` by default and provides AppContextSwitches to fallback to using `ClaimsIdentity`. See PR [#2977](https://github.com/AzureAD/microsoft-identity-web/pull/2977) for details.
+
+### Bug fixes:
+- For SN/I scenarios, Id Web's `GetTokenAcquirer` now sets `SendX5C`. See issue [#2887](https://github.com/AzureAD/microsoft-identity-web/issues/2887) for details.
+- Fix for Instance/Tenant parsing for V2 authority. See PR [#2954](https://github.com/AzureAD/microsoft-identity-web/issues/2954) for details.
+- Fix a format exception: `The input string " was not in a correct format` due to incorrect regex. See issue [#2879](https://github.com/AzureAD/microsoft-identity-web/issues/2879) for details.
+- Id Web now has an upper bound set on its dependency on Microsoft.Identity.Abstractions to version 7x. See PR [#2962](https://github.com/AzureAD/microsoft-identity-web/pull/2962) for details.
```

Comment:
```suggestion
- Microsoft.Identity.Web 3.1.0 now has an upper bound set on its dependency on Microsoft.Identity.Abstractions to version 7x to avoid you Microsoft.Identity.Abstractions 8.0.0. See PR [#2962](https://github.com/AzureAD/microsoft-identity-web/pull/2962) for details.
```